metadata-ingestion/src/datahub/ingestion/reporting/datahub_ingestion_run_summary_provider.py

```diff
@@ -149,7 +154,7 @@ def _get_recipe_to_report(self, ctx: PipelineContext) -> str:
         if not self.report_recipe or not ctx.pipeline_config._raw_dict:
             return ""
         else:
-            return json.dumps(ctx.pipeline_config._raw_dict)
+            return json.dumps(redact_raw_config(ctx.pipeline_config._raw_dict))
```

Comment:
Is this the only place we should redact the password fields, or was this PR about to redact the password from here?

Reply:
This PR was to redact the password showing up there, but will be useful if we ever need to write config to logs in the future as well.